shacl12-core/index.html

```diff
@@ -7737,8 +7737,8 @@ <h2>Security and Privacy Considerations</h2>
 		<section id="ack" class="appendix informative">
 			<h2>Acknowledgements</h2>
 			<p>
-				The original 1.0 version of SHACL was produced by the RDF Data Shapes Working Group.
-				See its <a href="https://www.w3.org/TR/shacl/#ack">SHACL 1.0 Acknowledgements section</a>.
+				The original SHACL core specification was produced by the RDF Data Shapes Working Group.
+				See its <a href="https://www.w3.org/TR/shacl/#ack">Acknowledgements section</a>.
```

Comment:
Since this is a back-in-time link, it would be better to use the dated URL.

```suggestion
				See its <a href="https://www.w3.org/TR/2017/REC-shacl-20170720/#ack">Acknowledgements section</a>.
```